server/room-battle.ts

```diff
@@ -1112,6 +1121,31 @@ export class RoomBattle extends RoomGames.RoomGame {
 		// @ts-ignore
 		this.room = null;
 	}
+	async getTeam(user: User) {
+		const id = user.id;
+		const player = this.playerTable[id];
+		if (!player) return;
+		void this.stream.write(`>requestteam ${player.slot}`);
+		const teamDataPromise = new Promise<string[]>(resolve => {
+			this.playerTeamResolver = resolve;
+		});
+		const resultStrings = await teamDataPromise;
+		const result = resultStrings.map(item => Dex.fastUnpackTeam(item))[0];
+		return result;
+	}
+	onChatMessage(message: string, user: User): string | false {
+		void this.stream.write(`>chat-inputlogonly ${user.getIdentity(this.room.roomid)}|${message}`);
+		return false;
```

Comment:
tbh this API should probably be the one where "no return value" means "don't do anything".

Returning `false` definitely makes me nervous.

Reply:
Oh, can i rewrite this part of the API then? I don't like how it handles anything other than strings.

Reply:
Hm, it depends on how hotpatchable an API change to `onChatMessage` would be. I might need to handle this myself.

Reply:
Mostly should be hotpatchable? 
All the games are hotpatchable, and so is CommandContext.canTalk

Reply:
I handled it.

Reply:
(You'll need to rebase, though.)